README.md

```diff
@@ -1,2 +1,54 @@
-# get-datetime-details-while-tapping-the-.NET-MAUI-Calendar
-This repository contains a sample of get datetime details while tapping the .NET MAUI Calendar.
+# How to get Datetime details while tapping the .NET MAUI Calendar
+
+This repository contains a sample to get the datetime details while tapping using the [Syncfusion .NET MAUI Calendar](https://help.syncfusion.com/maui/calendar/getting-started) control.
```

Comment:
This repository contains a sample to get the DateTime details while tapping using the [Syncfusion .NET MAUI Calendar](https://help.syncfusion.com/maui/calendar/getting-started) control.

Reply:
I have changed the content as you mentioned.